lnrpc/rpc.proto

```diff
@@ -222,9 +222,17 @@ message WalletBalanceResponse {
     double balance = 1;
 }
 
+message RTChannel {
```

Comment:
Changed name to RoutingTableLink
Yes, edgeID represents channel outpoint so i changed edgeID to outpoint.